Create schematics topic (#1604)
IgniteUI/igniteui-docfx: en/components/general/cli/auth-template.md

> Ignite UI CLI provides support of multiple versions of a project (called project templates) to choose from when using either the [`new` command](../cli-overview.md#commands) or the [Step by step mode](step-by-step-guide.md).

Reviewer: ```suggestion
There are multiple versions of a project (called project templates) to choose from when using either the [`new` command](../cli-overview.md#create-new-project) or the [Step by step mode](step-by-step-guide.md).
```
